VideoLocker/src/main/java/org/edx/mobile/model/course/CourseComponent.java

```diff
@@ -151,7 +151,7 @@ public void setResponsiveUI(boolean responsiveUI) {
     }
 
     public boolean isContainer(){
-        return children != null && children.size() > 0;
+        return !type.isUnit;
```

Comment:
Again, we can add a `null` check here, as suggested in the `BlockModel` section.